client/src/sections/resources/PowerfulGlove.tsx

```diff
@@ -31,13 +31,17 @@ const PowerfulGlove = () => {
       hide={!useHave($item`Powerful Glove`)}
     >
       {batteryUsed < 100 && (
-        <Line>{100 - batteryUsed}% charge remaining today.</Line>
-      )}
-      {batteryUsed < 95 && (
         <Line>
-          <Text as="span" color="gray.500">
-            {Math.floor((100 - batteryUsed) / 10)} shots of replace monster.
-          </Text>
+          {100 - batteryUsed}% charge{" "}
+          {batteryUsed <= 90 && (
+            <Text as="span" color="gray.500">
+              {`(${plural(
+                Math.floor((100 - batteryUsed) / 10),
+                "replace monster"
```

Comment:
this is super minor (like all my comments in this review lol) but should we change this to be "Can replace 10 monsters" instead of "10 replace monsters"? it just grammatically feels a little weird. agree we should cut this down just not sure of the best way to phrase it.